src/components/MenuDropdown/menu-dropdown.css

```diff
@@ -0,0 +1,12 @@
+.dropdown-menu {
+    width: 100%;
```

Comment:
Consider adding a few pixels of padding-top and padding-bottom to this rule, I think it looks a bit crowded when multiple categories are collapsed
![collapsible-dd](https://user-images.githubusercontent.com/8595776/66425094-db219980-e9dc-11e9-830e-db273218aa6e.png)

Reply:
definitely did. I added some padding for any .dropdown-menu that is also .closed